### ISSUE TITLE
Screen reader is not announcing as "selected" for selected date in calendar.

### ISSUE THREAD

**Issue by msftedad** [NuGet/docs.microsoft.com-nuget#3168]
**Repro steps:**

1. Launch the URL : [Angular powered Bootstrap (ng-bootstrap.github.io)](https://ng-bootstrap.github.io/#/components/datepicker/examples)
2. Turn on Narrator/NVDA/JAWS.
3. TAB till "Basic Date Picker".
4. TAB till the current date in calendar.
5. USE Arrow key to navigate to different dates.
6. Press enter on date to select it, selected date will get highlighted.
7. Observe that Screen reader is not announcing as "selected" for selected date in calendar.

**Actual Result:** 
Screen reader is not announcing as "selected" for selected date in calendar.

**Expected Result:**
After selecting any date from calendar, screen reader should announce the date as selected.

**Attachments:**

https://github.com/NuGet/docs.microsoft.com-nuget/assets/93735775/cd292924-4912-460e-b9ba-46cb61fd6758

https://github.com/NuGet/docs.microsoft.com-nuget/assets/93735775/3670c87b-0d91-4747-8a64-938ce29679c5

[Calendar date_JAWS.webm](https://github.com/NuGet/docs.microsoft.com-nuget/assets/93735775/bdf7356a-a7f8-4419-bd44-16dbf05930e5)

**Comment by zivkan** [comment]
This repo is for NuGet's official documentation.

You probably want to report this in https://github.com/ng-bootstrap/ng-bootstrap
